xmtp/src/storage/encrypted_store/models.rs

```diff
@@ -36,6 +42,7 @@ pub struct DecryptedMessage {
     pub convo_id: String,
     pub addr_from: String,
     pub content: Vec<u8>,
+    pub state: i32,
```

Comment:
Out of scope for now, but Decrypted Message will want to have a field 'install_from' to record which installation the message originated from.